spec/views/devise/sessions/new.html.slim_spec.rb

```diff
@@ -12,7 +12,7 @@
     render
 
     section = assert_select("h2")
-    expect(section.text.strip).to eq "Log in"
+    expect(section.text.strip).to eq "Sign In"
```

Comment:
Are we sure about the text? Usually it is either "Sign in" or "Log in" and not mixed. The last change in the file above is using "Log in" while here we use "Sign in".